src/client/Microsoft.Identity.Client/AppConfig/PopAuthenticationConfiguration.cs

```diff
@@ -0,0 +1,56 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Net.Http;
+using System.Net.Http.Headers;
+using Microsoft.Identity.Client.AuthScheme.PoP;
+
+namespace Microsoft.Identity.Client.AppConfig
+{
+    /// <summary>
+    /// Configuration properties used to construct a Proof of possesion request
+    /// </summary>
+    public class PopAuthenticationConfiguration
+    {
+        /// <summary>
+        /// An HTTP uri to the protected resource which requires a PoP token. The PoP token will be cryptographically bound to the request.
+        /// </summary>
+        public Uri RequestUri { get; set; }
+
+        /// <summary>
+        /// The HttP method to be bound to the request
```

Comment:
The HTTP method ("GET", "POST" etc.) method that will be bound to the token. Leave null and the POP token will not be bound to the method.